.pipelines/pipeline.user.windows.yml

```diff
@@ -129,6 +129,9 @@ build:
             - 'modules\launcher\Plugins\Microsoft.Plugin.Program\Telemetry.dll'
             - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\Microsoft.PowerToys.Run.Plugin.Registry.dll'
             - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\ManagedTelemetry.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.WindowsSettings\ManagedTelemetry.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.WindowsSettings\Microsoft.PowerToys.Run.Plugin.WindowsSettings.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.WindowsSettings\WindowsSettings.json'
```

Comment:
Not all dlls are added to signing process. Created an issue to solve it for all plugins https://github.com/microsoft/PowerToys/issues/11641